services/gateway/apis/http-version3/methods/transactionsEstimateFees.js

```diff
@@ -29,11 +29,12 @@ module.exports = {
 			optional: false,
 			type: 'object',
 			props: {
+				id: { optional: true, type: 'string', patter: regex.HASH_SHA256 },
```

Comment:
Also we need to add this to swagger and possibly API documentation, if applicable.
```suggestion
				id: { optional: true, type: 'string', pattern: regex.HASH_SHA256 },
```